src/py/flwr/client/middleware/utils.py

```diff
@@ -0,0 +1,37 @@
+# Copyright 2023 Flower Labs GmbH. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+"""Utility functions for middleware layers."""
```

Comment:
We could move the code in this module into the `flwr.client.flower` module

Reply:
I think it would be better if we can have a `middleware` directory to store all built-in middleware layers, like secaggplus_middleware.py. Thus I prefer to keep the utility function here. What's your view on this?

Reply:
Makes sense